src/main/java/com/transportcompany/transportcompany/models/dtos/UserDTO.java

```diff
@@ -10,6 +10,7 @@
 @ToString
 public class UserDTO implements Serializable {
 
+    @NotEmpty
     private long id;
```

Comment:
Do we need to provide an ID for the `createUser` API?
We need to be able to return in the response the `id` of the user for GET APIs, but this should be a mandatory field when we create a new user?